pkg/plugin/driver/base.go

```diff
@@ -66,6 +66,11 @@ func (b *Base) GetName() string {
 	return b.Definition.Name
 }
 
+// GetIOrder returns the order of this Job plugin.
```

Comment:
Hi @johnSchnake,

`// GetIOrder` -> `// GetOrder`

Also, consider rewriting the function signature as `func (b Base) GetOrder() int` since there is no reason for the function to be able to write any values to the address of `b`. A read on a copy will suffice and offers better protection.

Reply:
👍 on the rename.

As far as the pointer, I agree with the unencessity since it is a read. However, go best practice states that you shouldn't have a [mixture](https://tour.golang.org/methods/8) of pointer/value receivers for a given type. Since other pointer receiver methods exist (and I wasn't planning on changing them all, nor do I think I could with all the implications because this is used by all the other plugin types). (also fwiw I hate linking to the tour as a 'best practice' reference but I can't find the original link I'm thinking of. We can talk more about this if it is a serious concern to you)

Reply:
Hi @johnSchnake,

I'm not overly concerned about this, but as long as there's a clear pattern to when things read and when things write, I think it's fine. All of GoVmomi does it. Read functions use byCopy receivers, and write functions use byAddress receivers.

I don't feel strongly either way. Just noticed it.